### ISSUE TITLE
You can’t modify the line height or spacing of text without shifting the layout

### ISSUE THREAD

**Issue by sssoz** [oaworks/oa-works#125]
This fails [WCAG 2.1 success criterion 1.4.12: **text spacing**](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html).

**Comment by thinkbulecount2** [comment]
I suggest calculating the line-height as a relative measurement (i.e. either as em or percentage) depending on the font-size, if that makes sense. Based on WCAG 2.1's criterion:
```Line height (line spacing) to at least 1.5 times the font size;
Spacing following paragraphs to at least 2 times the font size;
Letter spacing (tracking) to at least 0.12 times the font size;
Word spacing to at least 0.16 times the font size.```

**Comment by thinkbulecount2** [comment]
Not sure what this means, @sssoz, can you explain?

**Comment by sssoz** [comment]
@thinkbulecount2 I can’t remember anymore actually. This can be closed, I suppose!
